### ISSUE TITLE
[4.0] "Help Server"-Dropdown useless, "Refresh"-Button don't work

### ISSUE THREAD

**Issue by ghost** [joomla/joomla-cms#26028]
### Steps to reproduce the issue
Open ```Global Configuration > System > System Settings```:  ```Help Server```has a Dropdown including one Value, the "Refresh"-Button don't work and give no Errors in Console.

![Bildschirmfoto 2019-08-26 um 06 27 50](https://user-images.githubusercontent.com/8235763/63665626-5c75f180-c7cc-11e9-9d58-df30408e6049.png)

#### System Information

Setting | Value
-- | --
PHP Built On | Linux lamp129.cloudaccess.net 3.10.0-962.3.2.lve1.5.24.5.el6h.x86_64 #1 SMP Thu Nov 22 10:07:47 EST 2018 x86_64
Database Type | mysql
Database Version | 5.7.23-cll-lve
Database Collation | utf8_general_ci
Database Connection Collation | utf8mb4_general_ci
PHP Version | 7.3.7
Web Server | Apache
WebServer to PHP Interface | cgi-fcgi
Joomla! Version | Joomla! 4.0.0-alpha11-dev Development [ Amani ] 29-June-2019 11:27 GMT
User Agent | Mozilla/5.0 (Macintosh; Intel Mac OS X 10.13; rv:68.0) Gecko/20100101 Firefox/68.0
Multilanguage | Yes: French, German, Persian

**Comment by infograf768** [comment]
In 3.9.7 was deleted the field type `helpsite` from user.xml and application.xml which contained the Refresh button, but the `HelpSiteField` is still present in `/libraries/src/Form/Field/HelpsiteField.php` in staging.

See https://github.com/joomla/joomla-cms/pull/24927

This was not ported back to 4.0 both in 
`/administrator/components/com_config/forms/application.xml`
and
`/administrator/components/com_users/forms/user.xml`
where the field type `helpsite` still exists and does not throw an error as the type still exists in
`/libraries/src/Form/Field/HelpsiteField.php`

Basically all this should be deleted from 4.0

**Comment by infograf768** [comment]
Will make a PR for that now.

**Comment by HLeithner** [comment]
3.9.7 has not been merged into 4.0 yet, you don't need to create a pr.

**Comment by infograf768** [comment]
Thanks. Did not remember that.

**Comment by ghost** [comment]
Thanks for comments resolving the issue.
